src/applications/hca/containers/IDPage.jsx

```diff
@@ -96,10 +96,11 @@ class IDPage extends React.Component {
 
     const { enrollmentStatus, noESRRecordFound, shouldRedirect } = this.props;
 
-    // Redirect to intro if a logged in user directly accessed this page.
-    if (shouldRedirect) this.props.router.push('/');
-
-    if (
+    // Redirect to intro if a logged in user directly accessed this page...
+    // ...otherwise handle the response from the ID Form
+    if (shouldRedirect) {
+      this.props.router.push('/');
+    } else if (
```

Comment:
Adding this as an `else` because we only want to prefill the HCA and navigate the user to the form _if they are logged out_. The function of the prefill for logged-out users is to simply take the info the user had entered into the ID form and transfer it to the HCA. But what was happening is when a user logged-in from this page, these two functions would fire and the `goToNextPage()` would direct the user to where they left off on the HCA form (if they had an in-progress HCA)